packages/client/src/constants/tab.ts

```diff
@@ -56,7 +56,13 @@ export const builtinTab: [string, ModuleBuiltinTab[]][] = [
       path: 'pinia',
       title: 'Pinia',
     },
-
+    {
+      icon: 'i-carbon:data-base',
```

Comment:
I didn't find any icon of Vuex.

Maybe there is a better one for it? instead of `data-base`